### ISSUE TITLE
how to training on COCO dataset

### ISSUE THREAD

**Issue by yang-haoxiang** [illusory2333/Adaptive-module-and-heatmap-translator#1]
I'd like to ask how to achieve results on the COCO dataset, or how to train on my own dataset.

**Comment by illusory2333** [comment]
1. how to achieve results on the COCO dataset
If you want to use your model to train on the COCO dataset, you can check out the https://github.com/HRNet/HRNet-Human-Pose-Estimation. 
2. how to train on my own dataset
You'll need to prepare a training dataset, validation dataset, and test dataset. Both training and validation datasets need to be labeled with human joint points. The image can then be normalized and trained in a neural network.
